ensure that __cmpxchg_double_slab.isra.45 and
__cmpxchg_double_slab.isra.45.part.46 aren't correlated.
Index: src/mm/slub.c
===================================================================
--- src.orig/mm/slub.c
+++ src/mm/slub.c
@@ -5320,6 +5320,9 @@ void get_slabinfo(struct kmem_cache *s,
 	int node;
 	struct kmem_cache_node *n;
 
+	if (!jiffies)
+		printk("slabinfo\n");
+
 	for_each_kmem_cache_node(s, node, n) {
 		nr_slabs += node_nr_slabs(n);
 		nr_objs += node_nr_objs(n);
